rpc/transaction_test.go

```diff
@@ -309,3 +309,30 @@ func TestDeployOrDeclareReceipt(t *testing.T) {
 
 	}
 }
+
+// TestGetTransactionStatus tests starknet_getTransactionStatus
+func TestGetTransactionStatus(t *testing.T) {
+	testConfig := beforeEach(t)
+
+	type testSetType struct {
+		TxnHash      *felt.Felt
+		ExpextedResp GetTxnStatusResp
+	}
+
+	testSet := map[string][]testSetType{
+		"mock": {},
+		"testnet": {
```

Comment:
This test does not run by default (only `"mock"` tests are executed, it seems). Are all "test environments" executed by the CI, or are only mock tests run?

Reply:
Ah I see it appears those tests are run by the CI :+1: